lib/htty/requests_util.rb

```diff
@@ -118,6 +118,6 @@ def self.headers_from(request)
   # starting from 2.0.0 net/http requires headers to be hashes
   require 'rubygems' # Not necessary under Ruby v2
   HEADERS_MUST_BE_AN_HASH =
-    Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.0.0')
+    Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.0.0')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.